jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3Session.java

```diff
@@ -216,7 +217,7 @@ private CompletableFuture<Void> goAway(GoAwayFrame frame)
             {
                 long error = HTTP3ErrorCode.REQUEST_CANCELLED_ERROR.code();
                 String reason = "go_away";
-                failStreams(stream -> true, error, reason, true);
+                failStreams(stream -> true, error, reason, true, new ClosedChannelException());
```

Comment:
Shouldn't this be a `QuietException` too?

Reply:
No because the quiet exceptions are used when receiving an event, while here it's generating the event.